### ISSUE TITLE
Fake Plates

### ISSUE THREAD

**Issue by Yaangosu** [ioShinigami/SoloRP-Directory#97]
Idk how you would go about doing this but being able to put fake plates on your vehicle would make more sense than "scratching plates". Makes it so you need to do some extra things if you are gonna use anything other than a local car for heists or racing.

**Comment by patchell13** [comment]
I agree. Would be a nice feature to be able to hide the identity of the car without a cop walking up to the vehicle and verifying the VIN or something similar.
